templates/keyword_result.html

```diff
@@ -0,0 +1,30 @@
+{{ define "content" }}
+
+<div class="result-keyword">
+  <div class="result-keyword__header">
+    <a class="result-keyword__google-link" href="/keyword/{{ .data.keyword.ID }}/html" target="_blank">
+      <img src="/dist/svg/google.svg">
```

Comment:
You are not using the link alt attribute and image replacement technique we discussed before (using `<span class="sr-only">View the cached HTML page</span>`, see the [Tailwind Docs](https://tailwindcss.com/docs/screen-readers)). 

HTML must be semantic and accessible 💪

Reply:
Decide to do this in another PR as I found some issue with ParcelJS
issue link: #41

Reply:
Oh. it's still the same issue as before 😔

Reply:
Added TODO in 0245d45c0d3d5de938135367e2dab1eb5e05c017

Reply:
Added link under the G icon in 392c9a7d0d6407b983884f07421656e2e6deed0b
![Screen Shot 2564-01-20 at 16 00 05](https://user-images.githubusercontent.com/29707647/105151452-915cb400-5b38-11eb-8b05-04bb78434381.png)